cmake/alpakaCommon.cmake

```diff
@@ -441,6 +452,12 @@ if(ALPAKA_ACC_GPU_CUDA_ENABLE)
                 alpaka_set_compiler_options(DEVICE target alpaka $<$<COMPILE_LANGUAGE:CUDA>:-Xptxas=-v>)
             endif()
 
+            if(ALPAKA_RELOCATABLE_DEVICE_CODE)
+                alpaka_set_compiler_options(DEVICE target alpaka $<$<COMPILE_LANGUAGE:CUDA>:-rdc=true>)
+            else()
```

Comment:
`elseif(ALPAKA_RELOCATABLE_DEVICE_CODE STREQUAL OFF)`

Alpaka is keeping always the default except we explicitly disable or enable such an option.